.travis.yml

```diff
@@ -10,17 +10,19 @@
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
 
+dist: xenial
```

Comment:
Since "trusty" seems to have gone out of support (https://forums.mysql.com/read.php?11,677237,677268#msg-677268), is there an issue just updating to "bionic" (18.04) instead of "xenial" (16.04)?

Reply:
xenial is current default, we used to be on trusty on which mysql became unsupported...

Reply:
...in the other words - I don't have time to play with it right now, will be away on Mon and then from Fri. This is mostly quick way to unblock people while I'm gone....

Reply:
I see. "trusty" wasnt declared anywhere as the distribution, right? Why wasnt "xenial" used then as the new default (seems to have been set as the default from May, 2019.